api/game.py

```diff
@@ -232,6 +293,28 @@ def get_player_info(player_names):
             good_roles_in_game.append("Tristan")
             good_roles_in_game.append("Iseult")
 
+    # if we took no lovers and not the older sibling:
+    # we must replace one role with a new role
+    # if there are not enough roles to replace, we remove the older sibling and one other role,
+    # and add the lovers
+    if (
+        sum(gr in ["Tristan", "Iseult"] for gr in good_roles_in_game) == 0
+        and "Older Sibling" not in good_roles_in_game
+        and num_good > 1
+    ):
+        available_roles = (
+            set(good_roles) - set(good_roles_in_game) - set(["Tristan", "Iseult", "Older Sibling"])
+        )
+        if len(available_roles) == 0:
+            good_roles_in_game.remove("Older Sibling")
```

Comment:
Didn't we just check that Older Sibling isn't in the goog_roldes_in_game? Also, if we add the lovers, don't we need to remove 2 of the currently used good_roles_in_game?

I feel like instead of randomly generating and then correcting, it might be simpler to decide if it's an Older Sibling or a Lovers game and then fill in the rest of the characters.

Reply:
i think we should just rewrite all of this code lmaoo